packages/common/src/compiler/translator.mock.ts

```diff
@@ -1297,145 +1297,7 @@ export const LIGHTDASH_TABLE_WITH_ADDITIONAL_DIMENSIONS: Omit<
             colors: undefined,
             index: 0,
             isAdditionalDimension: true,
-            isIntervalBase: true,
-        },
-        created_at_day: {
-            colors: undefined,
-            compact: undefined,
-            description: undefined,
-            fieldType: FieldType.DIMENSION,
-            format: undefined,
-            groups: ['Created at'],
-            hidden: true,
-            index: 0,
-            isAdditionalDimension: true,
-            isIntervalBase: false,
-            label: 'Created at day',
-            name: 'created_at_day',
-            requiredAttributes: undefined,
-            round: undefined,
-            source: undefined,
-            sql: "DATE_TRUNC('DAY', ${metadata}-->'created_at')",
-            table: 'myTable',
-            tableLabel: 'My table',
-            timeInterval: TimeFrames.DAY,
-            timeIntervalBaseDimensionName: 'created_at',
-            type: DimensionType.DATE,
-        },
-        created_at_month: {
-            colors: undefined,
-            compact: undefined,
-            description: undefined,
-            fieldType: FieldType.DIMENSION,
-            format: undefined,
-            groups: ['Created at'],
-            hidden: true,
-            index: 0,
-            isAdditionalDimension: true,
-            isIntervalBase: false,
-            label: 'Created at month',
-            name: 'created_at_month',
-            requiredAttributes: undefined,
-            round: undefined,
-            source: undefined,
-            sql: "DATE_TRUNC('MONTH', ${metadata}-->'created_at')",
-            table: 'myTable',
-            tableLabel: 'My table',
-            timeInterval: TimeFrames.MONTH,
-            timeIntervalBaseDimensionName: 'created_at',
-            type: DimensionType.DATE,
-        },
```

Comment:
Removed the mock for additional dimension intervals because in this case of the mock the base dimension is `string` and the additional dimension is `timestamp`, we no longer generate intervals additional dimensions in this case.